doc/sphinx/source/_static/empty

```diff
@@ -0,0 +1 @@
+This directory is empty.
```

Comment:
Does the _static directory need to exist in the repo, or can whatever puts stuff in here create it on the fly?